css/index.css

```diff
@@ -0,0 +1,352 @@
+
+body {
+  font-family: Ubuntu,sans-serif;
+}
+
+.pb-21 {
+  background-color: white;
+  padding-bottom: 21px;
+  border-bottom: 1px solid #cbcbcb;
+}
+
+#form-color {
+  background-color: white;
+}
+
+.icon-bar { 
+  background-color: black;
+  padding: 0px;
+  border: none;   
+}
+
+#pl-2 {
+  margin-top: 3px;
+}
+
+.navbar-toggle .icon-bar {
+  height: 1px;
+  width: 16px;
+}
+
+.btn-success {
+  width: 140px;
+  height: 37px;
+  margin-top: 14px;
+  border-radius: 18px;
+  background-color: #09b288;
+  font-weight: bold;
+  font-size: 18px;
+  margin-right: 10px;     
+}
+
+#img-banner {
+  width: 100%;
+  z-index: -1;
+  margin-top: 73px;
+  position: relative;    
+}
+
+#cs-logo {
+  width: 320px;
+  margin-top: 14px;
+  margin-left: 10px;
+}
+
+.ml-160 {
+  text-align: left;
+  width: 50%;
+  margin-left: 160px;
+}
+
+.mt-30 {
+  letter-spacing: 1px;
+  color: #492e87;
+  margin-top: 30px;
+}
+
+.text-black {
+  font-weight: 300;
+  color: #484848;
+}
+
+.pl-2 {
+  padding-top: 10px;
+}
+
+h1 {
+  font-weight: normal;
+  font-size: 37px;
+}
+
+.mt-20 {
+  font-size: 26px;
+  line-height: 40px;
+  letter-spacing: 1px;
+  font-weight: normal;
+  margin-bottom: -30px;
+}
+
+
+input[type=text], input[type=password] {
+  width: 325px;
+  height: 35px;
+  padding: 6px 6px;
+  margin: 12px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  border-radius: 6px;
+  text-align: left;
+  color: #8c8c8c;
+  font-weight: 200;
+  font-family: Ubuntu,sans-serif;
+  font-size: 13px;
+}
+
+
+#state {
```

Comment:
@indalgit use of id for css isn't a good idea... Do read about **why to use class for css instead of id** ?